tests/plugins/test_spaces_before_dots.py

```diff
@@ -0,0 +1,56 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+
+from pathlib import Path
+
+from tests.plugins import PluginTestCase
+from troubadix.plugins.spaces_before_dots import CheckSpacesBeforeDots
+
+
+class TestSpacesBeforeDots(PluginTestCase):
+
+    def test_ok(self):
+        nasl_file = Path("/some/fake/directory/test.nasl")
+        content = """
+              script_tag(name:"summary", value:"Foo Bar.");
+              script_tag(name:"solution", value:"meh.");
```

Comment:
```suggestion
              script_tag(name:"insight", value:"foo bar ...");
              script_tag(name:"solution", value:"meh.");
```

Might be valid so i would not report these

Reply:
tripple dots will not trigger a report. But included as a test in 9a8f9bdcf577ce0a5fc421742bfc464adbcfc1e4 to make sure.